contracts/src/v0.8/ccip/MultiAggregateRateLimiter.sol

```diff
@@ -32,27 +31,27 @@ contract MultiAggregateRateLimiter is IMessageInterceptor, AuthorizedCallers, IT
   /// @notice LocalRateLimitToken struct containing the local token address with the remote chain selector.
   /// The struct is used for removals and updates, since the local -> remote token mappings are scoped per-chain.
   struct LocalRateLimitToken {
-    uint64 remoteChainSelector; // ────╮ Remote chain selector for which to update the rate limit token mapping
-    address localToken; // ────────────╯ Token on the chain on which the multi-ARL is deployed
+    uint64 remoteChainSelector; // ─╮ Remote chain selector for which to update the rate limit token mapping.
+    address localToken; // ─────────╯ Token on the chain on which the multi-ARL is deployed.
   }
 
   /// @notice RateLimitTokenArgs struct containing both the local and remote token addresses.
   struct RateLimitTokenArgs {
-    LocalRateLimitToken localTokenArgs; // Local token update args scoped to one remote chain
-    bytes remoteToken; // Token on the remote chain (for OnRamp - dest, of OffRamp - source)
+    LocalRateLimitToken localTokenArgs; // Local token update args scoped to one remote chain.
+    bytes remoteToken; // Token on the remote chain, for OnRamp - dest, or OffRamp - source.
   }
 
   /// @notice Update args for a single rate limiter config update.
   struct RateLimiterConfigArgs {
-    uint64 remoteChainSelector; // ────╮ Remote chain selector to set config for
-    bool isOutboundLane; // ───────────╯ If set to true, represents the outbound message lane (OnRamp), and the inbound message lane otherwise (OffRamp)
+    uint64 remoteChainSelector; // ─╮ Remote chain selector to set config for.
+    bool isOutboundLane; // ────────╯ If set to true, represents the outbound message lane (OnRamp), and the inbound message lane otherwise (OffRamp)
```

Comment:
```suggestion
    bool isOutboundLane; // ────────╯ If set to true, represents the outbound message lane (OnRamp), and the inbound message lane otherwise (OffRamp).
```